addons/binding/org.openhab.binding.netatmo/README.md

```diff
@@ -447,16 +447,16 @@ Bridge netatmo:netatmoapi:home "Netatmo API" [ clientId="*********", clientSecre
 
 ```
 # Indoor Module
-Number Netatmo_Indoor_Temperature         "Temperature [%.2f °C]"          <temperature>      { channel = "netatmo:NAMain:home:inside:Temperature" }
-Number Netatmo_Indoor_Humidity            "Humidity [%d %%]"               <humidity>         { channel = "netatmo:NAMain:home:inside:Humidity" }
+Number:Temperature Netatmo_Indoor_Temperature         "Temperature [%.1f %unit%]"          <temperature>      { channel = "netatmo:NAMain:home:inside:Temperature" }
+Number:Dimensionless Netatmo_Indoor_Humidity            "Humidity [%d %unit%]"               <humidity>         { channel = "netatmo:NAMain:home:inside:Humidity" }
 Number Netatmo_Indoor_Humidex             "Humidex [%.1f °C]"              <temperature_hot>  { channel = "netatmo:NAMain:home:inside:Humidex" }
```

Comment:
If the humidex is a dimensionless quantity we should think about removing the `°C` in the example.

Reply:
Correct.